website/client/src/components/groups/membersModal.vue

```diff
@@ -655,7 +655,8 @@ export default {
         memberId: member._id,
         groupId: this.groupId,
       });
-      this.viewMembers();
+      if (this.invites.length === 0) this.viewMembers();
+      else this.viewInvites();
```

Comment:
is the else needed? I mean if there is at least one invite and viewMembers is not called it shouldn't switch at all?

Reply:
agreed, it is redundant! I've tested to make sure and updated my pull request to reflect that change. Thank you!